http-server/src/server.rs

```diff
@@ -186,15 +189,20 @@ impl Server {
 										log::error!("method_call: {} failed: {:?}", req.method, err);
 									}
 								} else {
-									send_error(req.id, &tx, ErrorCode::MethodNotFound);
+									let code = JsonRpcErrorCode::MethodNotFound;
+									send_error(
+										req.id,
+										&tx,
+										JsonRpcErrorObject { code, message: code.message(), data: None },
+									);
 								}
 							}
 							Err(_e) => {
-								let (id, err) = match serde_json::from_slice::<JsonRpcInvalidRequest>(&body) {
-									Ok(req) => (req.id, ErrorCode::InvalidRequest),
-									Err(_) => (None, ErrorCode::ParseError),
+								let (id, code) = match serde_json::from_slice::<JsonRpcInvalidRequest>(&body) {
+									Ok(req) => (req.id, JsonRpcErrorCode::InvalidRequest),
+									Err(_) => (None, JsonRpcErrorCode::ParseError),
 								};
-								send_error(id, &tx, err);
+								send_error(id, &tx, JsonRpcErrorObject { code, message: code.message(), data: None });
```

Comment:
Maybe we can have a `JsonRpcErrorCode::into_error_obj()` for this? `send_error(id, &tx, code.into_error_obj());`?